### PR TITLE
updated code to be a little more "sensor" universal and error capturing.

### PR DESCRIPTION
**Pull request by LazzaAU**
Breaking changes "IF" you've flashed a temperature sensor in the last two days. If so, lease reflash your temp sensor

### CODE REVIEW

File: TasmotaConfigs.py [hunk 1]
```diff
@@ -303,12 +304,15 @@ def getBacklogConfigs(self, location: str) -> list:
 			sensorValue = 'DewPoint'
 		cmds = list()
 		if 'envSensor' in self._deviceType:
-			if self.checkSensorBrand:
-				runConfigs = self.BACKLOG_TEMPSENSORCONFIGS
-			else:
+			if self.checkSensorBrand(): #if sensor is a listed temperature sensor then do this
+				self._rule2 = f'rule2 on tele-{self._brand}#temperature do var1 %value% endon on tele-{self._brand}#Humidity do var2 %value% endon on tele-{self._brand}#{sensorValue} do var3 %value% endon on tele-{self._brand}#{sensorValue} do event sendtemp endon on event#sendtemp do publish projectalice/devices/tasmota/feedback/{self._uid}/sensor {{"sensorBrand":"{self._brand}","sensorType":"temperatureSensor","siteId":"{location}","deviceType":"{self._deviceType}","Temperature":"%Var1%","Humidity":"%Var2%","{sensorValue}":"%Var3%","uid":"{self._uid}"}} endon '
```
Comment (philipp2310) [inline TasmotaConfigs.py:308]: save in constant like other rule2, outside of a function

Comment (philipp2310) [inline TasmotaConfigs.py:308]: can be stored in a constant like other rule2?

**Review by philipp2310 — COMMENTED** [review]
will try to extract the one constant (rule2) and merge later today

**Comment by LazzaAU** [comment]
yes agree a constant would make sense :)

**Comment by philipp2310** [comment]
will add the constant after pull directly - because github

**Pull request merged**